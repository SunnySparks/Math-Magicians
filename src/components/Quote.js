import 'bootstrap/dist/css/bootstrap.min.css';
import 'bootstrap/dist/js/bootstrap.bundle.min';
import '../App.css';
import React from 'react';

const Quote = () => (
  <div className="jumbotron">
    <div className="container">
      <div className="title-cont pt-5 pl-5">
        <h1>
          Quote of the day
        </h1>
      </div>
    </div>
    <nav className="container vertical-center column pt-5">
      <div className="p-5 text-cont">
        <h4>
          Mathematics as an expression of the human mind
          reflects the active will, the contemplative reason,
          and the desire for aesthetic perfection.
          Its basic elements are logic and intuition,
          analysis and construction, generality and individuality.
          â€” Richard Courant, German-American mathematician
        </h4>
      </div>
    </nav>
  </div>
);

export default Quote;
