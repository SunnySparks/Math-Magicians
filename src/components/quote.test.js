import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from './Quote';

it('Test Quote from Snapshot', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});

it('Test Quote from DOM', () => {
  render(<Quote />);
  expect(screen.getByText('Quote of the day')).toBeInTheDocument();
});
