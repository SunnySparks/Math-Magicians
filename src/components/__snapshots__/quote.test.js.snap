// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quote from Snapshot 1`] = `
<div
  className="jumbotron"
>
  <div
    className="container"
  >
    <div
      className="title-cont pt-5 pl-5"
    >
      <h1>
        Quote of the day
      </h1>
    </div>
  </div>
  <nav
    className="container vertical-center column pt-5"
  >
    <div
      className="p-5 text-cont"
    >
      <h4>
        Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. â€” Richard Courant, German-American mathematician
      </h4>
    </div>
  </nav>
</div>
`;
